autoload/preview_markdown.vim

```diff
@@ -100,8 +101,7 @@ function! preview_markdown#preview() abort
       redraw
     endif
 
-    let s:preview_buf_nr = term_start(parser, opt)
-    call delete(tmp)
+    let s:preview_buf_nr = term_start(cmd, opt)
```

Comment:
If you include the file in the command, remove the following options as they are no longer needed

```
          \ 'in_io': 'file',
          \ 'in_name': tmp,
```